### PR TITLE
Fix touch handler synthetic click

### PR DESCRIPTION
**Pull request by ilucin**
If `e.target` is an SVG element `click()` method doesn't exist. jQuery takes care of that.

### CODE REVIEW

**Comment by cibernox** [comment]
Nice edge case!
Can you add a regression test that covers this?

PS: I'm curious about what are you building with SVG

**Comment by ilucin** [comment]
> Nice edge case!

Yep. Good thing is the `click` method is invoked asynchronously so the error doesn't stop current event loop stack and everything still mostly works.

> Can you add a regression test that covers this?

Here it is. The test will fail if the fix is reverted.

> PS: I'm curious about what are you building with SVG

Nothing special, it's just an SVG icon inside of the trigger. This one in particular: http://take.ms/TeWjm :)

File: test-support/helpers/ember-basic-dropdown.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export function clickTrigger(scope, options = {}) {
 }
 
 export function tapTrigger(scope, options = {}) {
-  let selector = '.ember-basic-dropdown-trigger';
+  let selector = `.ember-basic-dropdown-trigger ${options.triggerChildSelector || ''}`.trim();
```
Comment (cibernox) [inline test-support/helpers/ember-basic-dropdown.js]: Instead of defining this ad-hoc option, you can use the `nativeTap` function I've just added to trigger a tap in the element inside.

Comment (cibernox) [inline test-support/helpers/ember-basic-dropdown.js]: Rebase over master to get that helper.

**Comment by cibernox** [comment]
Approved. Let's wait for travis.

**Comment by cibernox** [comment]
Thanks!
